dev/components/good-map.html

```diff
@@ -31,8 +31,12 @@
           this[name] = parseFloat(val);
           break;
         case 'map-options':
-          this.mapOptions = JSON.parse(val);
-          break
+          this.mapOptions = Object.assign(this.mapOptions, JSON.parse(val));
```

Comment:
This looks like it will forever accrue or otherwise merge in whatever values result from the new json parse, such that over time mapOptions could be filled with all kinds of stuff. Is that ok or should we do something like keep a clean base options that were set when map was originally created, and merge that into a separate unioned-options here?

It could be that this is fine, just raising for discussion while reading.

Reply:
True, this is a little messy. Fixed it, though interestingly enough this doesn't change the functionality, as setOptions on GoogleMap appends/overrides properties, but does not clear the existing ones.